genai/client_test.go

```diff
@@ -99,7 +104,7 @@ func TestLive(t *testing.T) {
 				iter := session.SendMessageStream(ctx, Text(msg))
 				for {
 					_, err := iter.Next()
-					if err == iterator.Done {
+					if errors.Is(err, iterator.Done) {
```

Comment:
Normally this is a good change. But we are following these [iterator guidelines](https://github.com/googleapis/google-cloud-go/wiki/Iterator-Guidelines#the-next-method), which guarantees that the error is actually `iterator.Done`. So using `==` here is fine.